tests/test_create_account.py

```diff
@@ -4,26 +4,29 @@
 import re
 
 import pytest
-from unittestzero import Assert
 
-from fxapom.fxapom import FxATestAccount
+from fxapom.fxapom import DEV_URL, FxATestAccount, PROD_URL
 
 
-@pytest.mark.nondestructive
 @pytest.mark.skip_selenium
 class TestCreateAccount(object):
 
-    def test_create_new_account_on_dev(self):
-        acct = FxATestAccount(base_url='https://www-dev.allizom.org').create_account()
-        Assert.true(acct.is_verified)
-        Assert.equal('https://stable.dev.lcip.org/auth/', acct.fxa_url)
+    def test_create_new_account_on_dev_by_default(self):
+        acct = FxATestAccount().create_account()
+        assert acct.is_verified
+        assert 'https://stable.dev.lcip.org/auth/' == acct.fxa_url
```

Comment:
I think this is related to the discussion we had about the weirdness of the code. I will change the test to not create an account, and we should also revisit the code in a separate issue to see if it can be improved.